src/utils/httpUtils.ts

```diff
@@ -0,0 +1,13 @@
+export function getURL(baseURL: string, path: string, orgId?: string): string {
+  if (routeToGateway(baseURL)) {
+    if (!orgId) {
+      throw new Error('Org is required for this operation');
+    }
+    return `${baseURL}/hidden/orgs/${orgId}/code${path}`;
+  }
+  return `${baseURL}${path}`;
```

Comment:
nit. I'd consider  using `path.join` here to avoid juggling with slashes, etc. Then it would not make any difference if you pass base url that ends with / and path that starts with /, eg:

```
path.join('https://snykgov.io/', '/test') => 'https://snykgov.io/test'
```
and 
```
path.join('https://snykgov.io', 'test') => 'https://snykgov.io/test'
```